clients/src/main/java/org/apache/kafka/common/requests/FetchRequest.java

```diff
@@ -44,10 +44,14 @@
     private static final String FETCH_OFFSET_KEY_NAME = "fetch_offset";
     private static final String MAX_BYTES_KEY_NAME = "max_bytes";
 
+    // default values for current version
+    public static final int DEFAULT_RESPONSE_MAX_BYTES = Integer.MAX_VALUE;
```

Comment:
I'm a little confused. Is this for older versions? If it's for the current version, shouldn't it match `FETCH_MAX_WAIT_MS_CONFIG` in `ConsumerConfig`?